src/main/java/command/Task.java

```diff
@@ -0,0 +1,32 @@
+package command;
```

Comment:
You can search up the rule of the convention for package in JAVA.
Now, it seems unmeaningful :)